devel/sealcurses/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           gitea 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+gitea.domain        git.skyjake.fi
+gitea.setup         skyjake sealcurses d64caa13bc06b68906bb0a5cc87f1896226b0918
+version             2022-05-23
+revision            0
+categories          devel
+platforms           darwin
```

Comment:
this is the default for `platforms` since MacPorts 2.7.0 - so the line can/should be removed (also in the other ports)